Website update - Vuepress

DavertMik opened the pull request:
Moved website from docusaurus to vuepress

* it is faster
* we can include more links into navigation
* more flexible in styling content
* has dropdowns fro helpers!

PeterNgTr commented on docs/.vuepress/config.js:
```diff
@@ -0,0 +1,254 @@
+      // { text: //   "books, link: //   "books" },
+      // { text: //   "videos, link: //   "videos" },
+      //   "examples"
+      // ]
```
should we remove those comment lines?

PeterNgTr commented on docs/.vuepress/theme/components/Footer.vue:
```diff
@@ -0,0 +1,96 @@
+      </div>
+    </section>
+    <div class="copyright">
+    <h5>CodeceptJS - supercharged end 2 end testing framework for NodeJS</h5>
```
`CodeceptJS - Supercharged E2E testing framework for NodeJS`

PeterNgTr commented on docs/.vuepress/theme/components/Home.vue:
```diff
@@ -0,0 +1,528 @@
+  <div class="row">
+    <div class="quote">
+      <p>
+        We have been using CodeceptJS as our UI testing framework, and it has made writing tests so simp  le for us, the amount of options and features available in CodeceptJS just out of the box are perfect for us to <b>test an application like Percona Monitoring and Management</b> (PMM), with so many dashboards & Metric plots.
```
`simp  le` not sure why we have space in between.

DavertMik commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631&bid=15637124)

DavertMik commented on runio.js:
```diff
@@ -35,21 +36,76 @@ module.exports = {
+      if (topic.slug === 'about-the-continuous-integration-category') continue;
+      body += `* ### [${topic.title}](https://codecept.discourse.group/t/${topic.slug}/)\n`;
+    }
+    writeToFile('docs/continuous-integration.md', cfg => cfg.line(body));
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected parentheses around arrow function argument.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631)

DavertMik replied:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected parentheses around arrow function argument.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631)

DavertMik replied:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected parentheses around arrow function argument.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631)

DavertMik commented on runio.js:
```diff
@@ -114,78 +173,110 @@ module.exports = {
       });
     }
-    await chdir('website/wiki', () => git(cfg => cfg.pull('origin master')));
+    await chdir('docs/wiki', () => git(cfg => cfg.pull('origin master')));
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected parentheses around arrow function argument.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631)

DavertMik replied:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected parentheses around arrow function argument.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631)

DavertMik replied:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected parentheses around arrow function argument.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631)

DavertMik commented:
I updated PR and moved out all website related things to https://github.com/codecept-js/website
So this PR actually simplifies development, as all styling and all static pages no longer require updates to main repo.

However, all docs/* still need to be formatted for vuepress and runio script was used to do that.

DavertMik commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631&bid=15637124)

DavertMik commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631&bid=15637124)
